### PR TITLE
User Story 1: Setup Gallery with Items

### PR DESCRIPTION
**Pull request by Long-HaHoang**
- Adding static JSON
- Adding a Title Header on top
- Mapping through a JSON to get items
- Setting up a gallery view of items
- Added temporary styling to the site

### CODE REVIEW

**Comment by JoergWolff** [comment]
LGTM

**Review by d-opri — APPROVED** [review]
looks good to me :)

File: pages/index.js [hunk 1]
```diff
@@ -1,3 +1,112 @@
...
+
+import products from "@/products.json";
+
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <AppContainer>
+      <ShopName>Shop Name</ShopName>
+      <GallerySection>
+        {/* TODO: refactor component in the next user story */}
```
Comment (F-Kirchhoff) [inline pages/index.js:11]: please leave no TODOs in your PR

File: pages/index.js [hunk 2]
```diff
@@ -1,3 +1,112 @@
...
+// Real styling comes much later
+const AppContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  border: solid blue;
+  height: auto;
+  width: 375px;
```
Comment (F-Kirchhoff) [inline pages/index.js:47]: You should not set the width of your app container to a fixed size, it should stay responsive. You can preview the app in the mobile view in your browser.

File: pages/index.js [hunk 3]
```diff
@@ -1,3 +1,112 @@
...
+  flex-direction: column;
+  place-items: center;
+  margin-top: 20px;
+`;
+
+const GalleryList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: wrap;
```
Comment (F-Kirchhoff) [inline pages/index.js:71]: you can delete this property

File: pages/index.js [hunk 4]
```diff
@@ -1,3 +1,112 @@
...
+  list-style: none;
+  gap: 10px;
+  margin: 0;
+  padding: 0;
+`;
+
+const Card = styled.li`
+  width: 328px;
+  height: 192px;
```
Comment (F-Kirchhoff) [inline pages/index.js:80]: You should not use fixed width and height values, this is not responsive

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Very nice Long, besides minor CSS suggesetions you are ready to go!

File: pages/index.js [hunk 5]
```diff
@@ -1,3 +1,108 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+import products from "@/products.json";
+
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <AppContainer>
```
Comment (d-opri) [inline pages/index.js:8]: im sure you will come back to this later on (i think you had a todo comment) but just case: you may want to create a Layout component, for a better overview. if you need a different layout for other pages you can also create a NestedLayout component and add a property getLayout to your page. [This is the Next.js page im referencing](https://nextjs.org/docs/basic-features/layouts)

**Review by d-opri — APPROVED** [review]

**Pull request merged**